### ISSUE TITLE
Suggestion - make Reading Glasses show enchantments on Enchanted Books

### ISSUE THREAD

**Issue by Stormwind99** [Nuchaz/BiblioCraft#578]
Make Reading Glasses show enchantments on Enchanted Books.    That way players with a library of enchanted books in bookcases can see what book they are looking for without opening every bookcase.

P.S.  Thanks for the fun mod!  Have you ever considered open sourcing BiblioCraft?  I'd make this change myself and submit a pull request to you if it was.  You could probably also attract help porting to new Forge and Minecraft versions.

**Comment by Nuchaz** [comment]
I'll have this working in the next release (v2.4.5). This actually used to work, but it looks like the way enchantments are stored on books changed (different NBT tag key).

I am thinking about going open source at some point, I just haven't made of my mind when.

**Comment by Nuchaz** [comment]
I just released v2.4.5 of BiblioCraft which fixed / re-added this feature.

**Comment by Stormwind99** [comment]
Thanks!

BTW, I made a couple open source mods that work well with Bibliocraft since I enjoyed the mapping and atlas features of Bibliocraft so much:

* [Palimpsest](https://github.com/Stormwind99/Palimpsest) - Erase Maps and Books for reuse by rubbing with Redstone Dust.
* [Pantography](https://github.com/Stormwind99/Pantography) - Copy Map item data to other Maps including different scale Maps.
   * Note: I'm considering making a pantography crafting table that would be required to copy map data between maps of different scales.

If you do open-source Bibliocraft, feel free to grab any of the logic/code/etc you find interesting or useful from those mods for your use.
